modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1911,6 +1911,48 @@ definition {
 		}
 	}
 
+	@description = "LPS-187081 - Verify the user cannot delete the only custom field in an object definition"
+	@priority = 5
+	test CannotDeleteSoleCustomField {
+		
+		task ("Given a custom object published with a custom field") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object",
+				objectName = "CustomObject",
+				pluralLabelName = "Custom Objects");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				enableShowFiles = "true",
+				fieldAttachment = "Upload Directly from the User's Computer",
+				fieldLabel = "Custom Attachment",
+				fieldType = "Attachment");
```

Comment:
Can you add the field using the API instead? This test doesn't require us to add the field using the UI